downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -310,21 +334,18 @@ rabbitmq:Client client = check new(”localhost”, 9090);
 ```
 
 ##### Security Updates
+
  - Removed encrypted passwords and hashed passwords support for Basic Auth file user store authentication.
 
 #### Bug Fixes
 
-To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Alpha4%22+label%3AType%2FBug).
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha4](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Alpha4%22+label%3AType%2FBug).
 
 ### Code to Cloud Updates
```

Comment:
All the links in Bug fixes are empty. So do we need this  section as no improvements or bug fixes? @anuruddhal ?

Reply:
No need if there aren't any bug fixes.

Reply:
We haven't done any bug fixes for this release. Let's remove this.

Reply:
Removed. Thanks a lot!